src/pagination/pagination.component.ts

```diff
@@ -231,11 +231,12 @@ export class Pagination {
 	 * @memberof Pagination
 	 */
 	get lastPage(): number {
-		return Math.ceil(this.model.totalDataLength / this.model.pageLength);
+		let last = Math.ceil(this.model.totalDataLength / this.model.pageLength);
```

Comment:
Not changing it later so
```suggestion
		const last = Math.ceil(this.model.totalDataLength / this.model.pageLength);
```